templates/stamp/stamp.json

```diff
@@ -0,0 +1,690 @@
+            }
+        },
+        "vmssUbuntuOs": {
+            "type": "string",
+            "defaultValue": "14.04.0-LTS",
+            "allowedValues": [
+                "16.04.0-LTS",
+                "14.04.0-LTS",
+                "12.04.5-LTS"
+            ],
+            "metadata": {
+                "description": "The Azure Ubuntu Image version to use for VMSS instances"
+            }
+        },
+        "mysqlUbuntuOs": {
+            "type": "string",
+            "defaultValue": "16.04.0-LTS",
+            "allowedValues": [
+                "16.04.0-LTS",
+                "14.04.0-LTS",
+                "12.04.5-LTS"
+            ],
+            "metadata": {
+                "description": "The Azure Ubuntu Image version to use for Mysql instances"
+            }
+        },
+        "mongoUbuntuOs": {
+            "type": "string",
+            "defaultValue": "16.04.0-LTS",
+            "allowedValues": [
+                "16.04.0-LTS",
+                "14.04.0-LTS",
+                "12.04.5-LTS"
+            ],
+            "metadata": {
+                "description": "The Azure Ubuntu Image version to use for Mongo instances"
+            }
+        },
+        "defaultUbuntuOs": {
+            "type": "string",
+            "defaultValue": "16.04.0-LTS",
+            "allowedValues": [
+                "16.04.0-LTS",
+                "14.04.0-LTS",
+                "12.04.5-LTS"
+            ],
+            "metadata": {
+                "description": "The default Azure Ubuntu Image version to use for VMs"
+            }
+        },
+        "jumpBoxDataDiskSizeGB": {
+            "type": "int",
+            "defaultValue": 50,
+            "metadata": {
+                "description": "The size of data disk to provision and attach to the VM"
+            }
+        },
+        "mongoNodeDataDiskSizeGB": {
+            "type": "int",
+            "defaultValue": 256,
+            "metadata": {
+                "description": "The size of data disk to provision and attach to the VM"
+            }
+        },
+        "mysqlNodeDataDiskSizeGB": {
+            "type": "int",
+            "defaultValue": 256,
+            "metadata": {
+                "description": "The size of data disk to provision and attach to the VM"
+            }
+        }
+    },
+    "variables": {
+        "defaultApiVersion": "2015-05-01-preview",
+        "apiVersion": "2016-09-01",
+        "availabilitySetApiVersion": "2015-06-15",
+        "networkApiVersion": "2016-03-30",
+        "computeApiVersion": "2016-03-30",
+        "insightsApiVersion": "2015-04-01",
+
+        "virtualMachineExtensionsApiVersion": "[variables('defaultApiVersion')]",
+        "virtualMachineApiVersion": "[variables('defaultApiVersion')]",
+        "storageApiVersion": "[variables('defaultApiVersion')]",
+
+        "storageAccountName": "[concat(parameters('clusterName'), 'vhdsa')]",
+        "storageAccountType": "Standard_LRS",
```

Comment:
Premium is ok but it's a question of SSD vs HDD and of course the associated costs. Do we have numbers suggesting we are exceeding the IOPS limits on Standard and thus actually need Premium storage? We certainly can go with Premium at a higher cost but if standard can do it, why incur the extra cost?

- [ ] Re-evaluate Standard_LRS vs Premium_LRS